llvm/lib/Target/NVPTX/MCTargetDesc/NVPTXInstPrinter.cpp

```diff
@@ -95,228 +96,262 @@ void NVPTXInstPrinter::printOperand(const MCInst *MI, unsigned OpNo,
 }
 
 void NVPTXInstPrinter::printCvtMode(const MCInst *MI, int OpNum, raw_ostream &O,
-                                    const char *Modifier) {
+                                    const char *M) {
```

Comment:
I think I misunderstood the request to use `StringRef`, to mean that those changes had to be part of this PR (my preference would have been to do those in different PRs..).

I didn't change the function signature in this PR because that would have pulled in a bit more changes into this PR (and its not straightforward due to how these APIs interact). My preference is to do that in a different PR.

Reply:
> my preference would have been to do those in different PRs..

+1 to splitting off independent changes into separate patches. Send them my way.